.circleci/config.yml

```diff
@@ -8,28 +8,52 @@ jobs:
             - checkout
 
             - restore_cache:
-                key: cache-{{ .Branch }}-{{ checksum "Pipfile.lock" }}
-            - run: pipenv install --deploy --dev
+                key: cache-{{ .Branch }}-{{ checksum "requirements.txt" }}
+            - run:
+                name: Install dependencies
+                command: |
+                  python3 -m venv venv
```

Comment:
AFAIK there's no need to run things inside a venv on CI, you can install direct to the main environment, which makes things look a little nicer.

Reply:
True. I'm too used to doing this anyway. It does have the advantage that it's most likely to be close to what contributors are doing.